Tests/GitBuddyTests/TestHelpers/ReleaseNotesJSON.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ReleaseNotesJSON.swift
+//  
+//
+//  Created by Antoine van der Lee on 13/06/2023.
+//
+
+import Foundation
+
+// swiftlint:disable line_length
+let ReleaseNotesJSON = """
```

Comment:
Was a bit surprised to see this uppercased.
```suggestion
let releaseNotesJSON = """
```